tests/functional/s/spelling/spelling.txt

```diff
@@ -0,0 +1,20 @@
+wrong-spelling-in-comment:1:0:None:None::"Wrong spelling of a word 'docstring' in a comment:
```

Comment:
I think it's the same on 3.7, 3.8, 3.9.. Probably because we're passing the line as an int instead of a node.

Reply:
Do we want to fix this in a follow-up PR? Or in this PR?

Reply:
I want to do two:
- Pass node instead of line number as int
- Add pylint's msgids and symbols to the dictionary automatically
Probably better after we merge this one and #5929 though, we're very bsy right now 😄

Reply:
Agreed! Could you make an issue about passing `node` instead of `line` in this checker though? So we don't forget about it!

Reply:
#6416